Author: Daniel Baumann <daniel.baumann@progress-technologies.net>
Description: Avoid overwriting build environment rather than to just extend it.

diff -Naurp dmidecode.orig/Makefile dmidecode/Makefile
--- dmidecode.orig/Makefile	2011-10-28 21:49:16.510949714 +0200
+++ dmidecode/Makefile	2012-01-21 11:05:50.109410854 +0100
@@ -13,7 +13,7 @@
 #
 
 CC      = gcc
-CFLAGS  = -W -Wall -Wshadow -Wstrict-prototypes -Wpointer-arith -Wcast-qual \
+CFLAGS += -W -Wall -Wshadow -Wstrict-prototypes -Wpointer-arith -Wcast-qual \
           -Wcast-align -Wwrite-strings -Wmissing-prototypes -Winline -Wundef
 #CFLAGS += -DBIGENDIAN
 #CFLAGS += -DALIGNMENT_WORKAROUND
@@ -23,7 +23,7 @@ CFLAGS += -O2
 #CFLAGS += -g
 
 # Pass linker flags here
-LDFLAGS =
+#LDFLAGS =
 
 DESTDIR =
 prefix  = /usr/local
